api/system-dhcp/scanNetwork

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
```

Comment:
Fix the year.

Also the whole implementation should go under the the `read` API with a new `scan` action (it seems the whole API should be refactored for handling multiple actions).
If you prefer to have the code in another script, invoke the helper from within read API.

(Please also note that this script is not executable inside the RPM, I had to manually fix it) 

See https://nethserver.github.io/nethserver-cockpit/api_guidelines/